keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountCredentialResource.java

```diff
@@ -231,4 +255,11 @@ public void setConfirmation(String confirmation) {
 
     }
 
+    private boolean credentialOwnedByUser(RealmModel realm, UserModel user, String credentialId) {
+        List<CredentialModel> models = session.userCredentialManager().getStoredCredentials(realm, user);
+        if (models.stream().filter( c -> c.getId() == credentialId).count() == 0) {
```

Comment:
You can just `return models.stream().....count()==0;`

Reply:
Good remark ... :)